.github/workflows/cut-release.yaml

```diff
@@ -0,0 +1,80 @@
+name: Cut Mod Release ⭐
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
+        description: 'Semver Bump Type'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+      binary_source:
+        description: 'Binary Source'
+        required: true
+        default: 'modbase'
+        type: choice
+        options:
+          - vanilla
+          - modbase
+          - build_binaries
```

Comment:
Since `LL-OpenGOAL-ModBase` has some c++ differences from `modbase` I think you'll always want to use the `build_binaries` option here